modules/aggregations/src/main/java/org/elasticsearch/aggregations/AggregationsPainlessExtension.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.aggregations;
+
+import org.elasticsearch.aggregations.pipeline.MovingFunctionScript;
+import org.elasticsearch.painless.spi.PainlessExtension;
+import org.elasticsearch.painless.spi.PainlessTestScript;
+import org.elasticsearch.painless.spi.Whitelist;
+import org.elasticsearch.painless.spi.WhitelistLoader;
+import org.elasticsearch.script.ScriptContext;
+import org.elasticsearch.search.aggregations.pipeline.MovingFunctions;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Extends the painless whitelist for the {@link MovingFunctionScript} to include {@link MovingFunctions}.
+ */
+public class AggregationsPainlessExtension implements PainlessExtension {
+    private static final Whitelist MOVING_FUNCTION_WHITELIST = WhitelistLoader.loadFromResourceFiles(
```

Comment:
From the neutral naming initiative, we're trying to eschew the term `whitelist`.  I'm having a hard time finding the reference right now, but I think we settled on `allowlist` instead? At any rate, renaming the class is probably outside the scope of this PR, but we can name the instance and resource file appropriately, and open a separate PR for the class.

Reply:
Yeah. I certainly thought about it while I was working on this but the painless class still has the old style name. I'll rename the things around it though.